custom/_legacy/mvp_docs/pillows.py

```diff
@@ -123,31 +123,35 @@ def process_indicators(self, namespaces, domain, doc_dict):
 
         for xform_id in xform_ids:
             try:
-                with IndicatorXForm.get_locked_obj(_id=xform_id) as xform_doc:
-                    if not xform_doc.xmlns:
-                        continue
-                    related_xform_defs = []
-                    for namespace in namespaces:
-                        related_xform_defs.extend(
-                            CaseDataInFormIndicatorDefinition.get_all(
-                                namespace,
-                                domain,
-                                xmlns=xform_doc.xmlns
-                            )
+                # first try to find the xform in the new db
+                xform_doc = IndicatorXForm.get_locked_obj(_id=xform_id)
```

Comment:
have you tested this? it looks like `get_locked_obj` returns a `LockManager` so i'm actually surprised that it works without the `with` statement.

the simplest fix would just be to use `IndicatorXForm.get()` which won't lock the document, but maybe that's not a big deal since it's not the real form.